java/arcs/android/devtools/JsonEncodings.kt

```diff
@@ -0,0 +1,66 @@
+package arcs.android.devtools
```

Comment:
We'll need the copyright comment here...

Reply:
Added here and elsewhere.